components/builder-jobsrv/habitat/default.toml

```diff
@@ -1,4 +1,5 @@
 log_level = "info"
+log_path = "/tmp"
```

Comment:
Should this be configurable? For things like this I typically hardcode it in the `config.toml` as `{{pkg.svc_log_path}}` like we did above for `key_dir = "{{pkg.svc_files_path}}"`

Reply:
Yeah, it's configurable right now because we still haven't resolved the directory permissions issue for the datadog agent (and potentially logrotate) to have access to files that are within the svc hierarchy.  Once those issues are resolved we can go back to hard coding this.